acinclude.m4

```diff
@@ -94,6 +94,33 @@ AC_DEFUN([MK_AM_LIBMAXMINDDB], [
   echo ""
 ])
 
+AC_DEFUN([MK_AM_GEOIP], [
+  echo "> checking for dependency: geoip"
+
+  AC_ARG_WITH([geoip],
+              [AS_HELP_STRING([--with-geoip],
+                [GeoIP OLD API library @<:@default=check@:>@])
+              ],
+              [
+                CPPFLAGS="$CPPFLAGS -I$withval/include"
+                LDFLAGS="$LDFLAGS -L$withval/lib"
+              ],
+              [])
+
+  mk_not_found=""
+  AC_CHECK_HEADERS(GeoIP.h, [], [mk_not_found=1])
+  AC_CHECK_LIB(geoip, GeoIP_open, [], [mk_not_found=1])
+
+  if test "$mk_not_found" = "1"; then
+    AC_MSG_WARN([Failed to find dependency: geoip])
+    echo "    - to install on Debian: sudo apt-get install libgeoip1 libgeoip-dev geoip-bin"
```

Comment:
I guess `geoip-bin` is not needed and `libgeoip1` is implied by `libgeoip-dev`, correct?